packages/moonstone/Divider/Divider.js

```diff
@@ -84,21 +75,73 @@ const DividerBase = kind({
 		className: ({spacing, styler}) => styler.append(spacing)
 	},
 
-	render: ({children, ...rest}) => {
-		delete rest.spacing;
+	render: (props) => {
+		delete props.spacing;
 
 		return (
-			// TODO: change to `marqueeOn="render"`
-			<MarqueeH3 marqueeOn="hover" {...rest}>{children}</MarqueeH3>
+			<h3 {...props} />
 		);
 	}
 });
 
-const Divider = Pure(
-	Skinnable(
-		DividerBase
-	)
+/**
+ * Moonstone-specific divider behaviors to apply to [Divider]{@link moonstone/Divider.DividerBase}.
+ *
+ * @hoc
+ * @memberof moonstone/Divider
+ * @mixes i18n/Uppercase.Uppercase
+ * @mixes moonstone/MarqueeDecorator.MarqueeDecorator
+ * @mixes ui/Skinnable.Skinnable
+ */
+const DividerDecorator = compose(
+	setPropTypes({
+		marqueeOn: PropTypes.oneOf(['hover', 'render'])
+	}),
+	defaultProps({
+		casing: 'word',
+		marqueeOn: 'render'
+	}),
+	Pure,
+	Uppercase,
+	MarqueeDecorator,
+	Skinnable
 );
 
+/**
+ * A Moonstone-styled divider with built-in support for uppercasing, and marqueed text.
+ *
+ * @class Divider
+ * @extends moonstone/Divider.DividerBase
+ * @memberof moonstone/Divider
+ * @mixes moonstone/Divider.DividerDecorator
+ * @ui
+ * @public
+ */
+
+/**
+ * Configures how the `children` string will be capitalized. By default, each word is capitalized.
+ *
+ * @see i18n/Uppercase#casing
```

Comment:
Without an `@name` field, there's no way for the documentation to know what to call this property.